consensus/ethash/consensus.go

```diff
@@ -68,7 +68,7 @@ var (
 // codebase, inherently breaking if the engine is swapped out. Please put common
 // error types into the consensus package.
 var (
-	errZeroBlockTime     = errors.New("timestamp equals parent's")
+	errOlderBlockTime    = errors.New("header timestamp is older or same as parent's")
```

Comment:
Perhaps `timestamp older than parent`. I know it's not as precise, but I kind of prefer shorter errors than long winded wording that doesn't add significantly more information.

Reply:
Updated